### PR TITLE
BXMSDOC-8105 Update Getting Started with process services

### PR DESCRIPTION
**Pull request by emmurphy1**
HTML draft:
http://file.ork.redhat.com/~emmurphy/BXMSDOC-8105/#assembly-getting-started-process-services

### CODE REVIEW

File: doc-content/drools-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc [hunk 1]
```diff
@@ -47,7 +47,7 @@ image::getting-started/bus-rule-task.png[]
 . Configure the business rule task:
 
 .. Click the business rules task.
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/drools-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc]: typo
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/drools-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc [hunk 2]
```diff
@@ -86,19 +86,19 @@ image::getting-started/exclusive_gateway.png[]
 
 . Create an outgoing connection from the exclusive gateway to a new business rule task:
 .. Click the exclusive gateway and click the *Create Task* icon.
-.. Drag the new task below the exclusive gateway as shown below.
+.. Drag the new task below the exclusive gateway as shown in the following image.
 .. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/new_br_task.png[]
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/drools-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc]: typo
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/drools-docs/src/main/asciidoc/GettingStartedDecServices/creating-users-proc.adoc [hunk 3]
```diff
@@ -24,7 +24,7 @@ In {CENTRAL}, you can use groups and roles to control permissions for a collecti
 . Click the *Groups* tab and click *Add to groups*.
 . Select `approver` and `kie-server`, and click *Add to selected groups*.
 . Click *Save*.
-. Select `Bill` from the user list, click *Edit* -> *Roles* -> *Add roles*.
+. Select `Bill` from the user list and click *Edit* -> *Roles* -> *Add roles*.
```
Comment (kaldesai) [inline doc-content/drools-docs/src/main/asciidoc/GettingStartedDecServices/creating-users-proc.adoc:27]: In the doc preview **_Click the Users tab,_** is present at the beginning of this step. I have attached the screenshot. Which one is correct?
```suggestion
. Click the *User* tab, select `Bill` from the user list and click *Edit* -> *Roles* -> *Add roles*.
```
image

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 4]
```diff
@@ -47,7 +47,7 @@ image::getting-started/bus-rule-task.png[]
 . Configure the business rule task:
 
 .. Click the business rules task.
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/enterprise-only/getting-started/business-process-proc.adoc]: typo
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/enterprise-only/getting-started/business-process-proc.adoc [hunk 5]
```diff
@@ -86,19 +86,19 @@ image::getting-started/exclusive_gateway.png[]
 
 . Create an outgoing connection from the exclusive gateway to a new business rule task:
 .. Click the exclusive gateway and click the *Create Task* icon.
-.. Drag the new task below the exclusive gateway as shown below.
+.. Drag the new task below the exclusive gateway as shown in the following image.
 .. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/new_br_task.png[]
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/enterprise-only/getting-started/business-process-proc.adoc]:
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedBusProc/business-process-proc.adoc [hunk 6]
```diff
@@ -47,7 +47,7 @@ image::getting-started/bus-rule-task.png[]
 . Configure the business rule task:
 
 .. Click the business rules task.
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedBusProc/business-process-proc.adoc]:
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedBusProc/business-process-proc.adoc [hunk 7]
```diff
@@ -87,19 +87,19 @@ image::getting-started/exclusive_gateway.png[]
 
 . Create an outgoing connection from the exclusive gateway to a new business rule task:
 .. Click the exclusive gateway and click the *Create Task* icon.
-.. Drag the new task below the exclusive gateway as shown below.
+.. Drag the new task below the exclusive gateway as shown in the following image.
 .. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/new_br_task.png[]
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedBusProc/business-process-proc.adoc]:
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc [hunk 8]
```diff
@@ -47,7 +47,7 @@ image::getting-started/bus-rule-task.png[]
 . Configure the business rule task:
 
 .. Click the business rules task.
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc]:
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc [hunk 9]
```diff
@@ -86,19 +86,19 @@ image::getting-started/exclusive_gateway.png[]
 
 . Create an outgoing connection from the exclusive gateway to a new business rule task:
 .. Click the exclusive gateway and click the *Create Task* icon.
-.. Drag the new task below the exclusive gateway as shown below.
+.. Drag the new task below the exclusive gateway as shown in the following image.
 .. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/new_br_task.png[]
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc]:
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc [hunk 10]
```diff
@@ -86,19 +86,19 @@ image::getting-started/exclusive_gateway.png[]
...
 .. In the *Properties* panel, enter `Retract Validation` in the *Name* field.
 .. Expand *Implementation/Execution*, select *New* from the *Rule Flow Group* menu, and input `error`.
 
 . Configure the connection between the exclusive gateway and the business rule task:
 .. Click the connection.
 +
 image::getting-started/created-connection.png[]
-.. If the *Properties* panel is not open, in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right cormer.
```
Comment (kaldesai) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc]:
```suggestion
.. If the *Properties* panel is not open, click the *Properties* image:getting-started/diagram_properties.png[] icon in the upper-right corner.
```

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc [hunk 11]
```diff
@@ -129,7 +129,7 @@ image::getting-started/user_task.png[]
 image::getting-started/correct-data-io.png[Screen capture of the Correct Data Data I/O assignments]
 .. Above the canvas, click *Save* to confirm your changes.
```
Comment (kaldesai) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc]: This is fluff. You can remove this. :) 
```suggestion
.. Above the canvas, click *Save*. 
```

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc [hunk 12]
```diff
@@ -187,10 +187,10 @@ image::getting-started/approval-io.png[Screen capture of the Final Approval Data
 . Above the canvas, click *Save* to confirm your changes.
 
 == Increasing the down payment
-The increasing the down payment business process checks to see if the applicant qualifies for the loan by increasing their down payment. The final result is either the final loan approval, or loan denial based on the applicant's inability to increase the down payment.
+The Increasing the Down Payment business process checks to see if the applicant qualifies for the loan by increasing their down payment. The final result is either the final loan approval, or loan denial based on the applicant's inability to increase the down payment.
```
Comment (kaldesai) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc:190]: Is **_Increasing the Down Payment_** the name of the business process?

File: doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-dmn-bpmn-maven-create.adoc [hunk 13]
```diff
@@ -16,9 +16,9 @@ mvn archetype:generate \
     -DarchetypeVersion={MAVEN_ARTIFACT_VERSION}
 ----
 +
-This command generates a Maven project with required dependencies and generates required directories and files to build your business application. You can set up and use Git version-control system (recommended) when developing a project.
+This command generates a Maven project with required dependencies and generates required directories and files to build your business application. You can use the Git versio control system (recommended) when developing a project.
```
Comment (kaldesai) [inline doc-content/shared-kie-docs/src/main/asciidoc/Install/proc-dmn-bpmn-maven-create.adoc]:
```suggestion
This command generates a Maven project with required dependencies and generates required directories and files to build your business application. You can use the Git version control system (recommended) when developing a project.
```

**Review by kaldesai — COMMENTED** [review]
Hey @emmurphy1, I have added my suggestions and few questions. Otherwise, your content looks good to me.

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc (hunk 12 above)
Comment (emmurphy1) [reply doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc:190]: Yes

**Pull request merged**